session/main_test.go

```diff
@@ -39,6 +38,9 @@ import (
 
 var testDataMap = make(testdata.BookKeeper, 1)
 
+var _ = flag.Bool("with-tikv", false, "workaroundGoCheckFlags: with-tikv")
+var _ = flag.String("pd-addrs", "", "workaroundGoCheckFlags: pd-addrs")
```

Comment:
We should remove these lines after removing corresponding tasks in PingCAP-QE/ci.

Reply:
OK, I will refactor the CI script recently.

Reply:
https://github.com/PingCAP-QE/ci/pull/975